gfx/portraits/portrait_modifiers/01_cloaks.txt

```diff
@@ -156,6 +156,50 @@ cloaks = {
 			}			
 		}
 	}
+	
+	# Warcraft - Warden's Cloak
+	fp1_warden_cloak = {
+		dna_modifiers = {
+			accessory = {
+				mode = add
+				
+				gene = cloaks
+				template = fp1_warden_cloak
+				range = { 0 1 }
+			}
+		}	
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				portrait_fp1_warden_clothing_trigger = yes
+				culture = { has_cultural_pillar = heritage_darnassian }
```

Comment:
I belive you don't need this trigger, the culture limit is handled in `portrait_fp1_warden_clothing_contents_trigger`.